client/modules/IDE/components/Timer.js

```diff
@@ -0,0 +1,40 @@
+import React, { PropTypes } from 'react';
+import moment from 'moment';
+
+class Timer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.showSavedTime = this.showSavedTime.bind(this);
+  }
+
+  componentWillMount() {
+    if (this.interval) {
+      clearInterval(this.interval);
+    }
+  }
+
+  componentDidMount() {
+    this.interval = setInterval(() => this.forceUpdate(), 10000);
+  }
+
+  showSavedTime() {
+    if (Math.abs(moment().diff(this.props.projectSavedTime)) < 10000) {
```

Comment:
I think this makes sense. Autosave runs every 30 seconds and is on by default, so I think it makes sense to have some time steps in between 0 and 30 seconds.